Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -263,13 +260,40 @@ extension SPSettingsViewController {
         return alert
     }
 
+    private func attemptPasskeyRegistration(for email: String, password: String) async {
+        do {
+            try await self.registerPasskey(for: email, password: password)
+            presentPasskeyRegistrationAlert(succeeded: true)
+        } catch {
+            NSLog("[PasskeyRegistration] Could not register passkey: %@", error.localizedDescription)
+            presentPasskeyRegistrationAlert(succeeded: false)
+        }
+        self.removeActivityIndicator()
```

Comment:
This invocation prob. belongs elsewhere (this method doesn't display the Spinner, but it assumes it's already up there)